content/en/tracing/setup/php.md

```diff
@@ -210,20 +213,70 @@ DD_TRACE_DEBUG=true php -S localhost:8888
 | `DD_DISTRIBUTED_TRACING`             | `true`      | Whether to enable distributed tracing                                       |
 | `DD_INTEGRATIONS_DISABLED`           | `null`      | CSV list of disabled extensions; e.g., `curl,mysqli`                        |
+| `DD_PRIORITY_SAMPLING`               | `true`      | Whether to enable priority sampling.                                        |
 | `DD_SAMPLING_RATE`                   | `1.0`       | The sampling rate for the traces. Between `0.0` and `1.0` (default)         |
 | `DD_SERVICE_NAME`                    | ``          | The default app name                                                        |
+| `DD_TRACE_AGENT_ATTEMPT_RETRY_TIME_MSEC` | `5000`  | IPC-based configurable circuit breaker retry time (in milliseconds)         |
+| `DD_TRACE_AGENT_CONNECT_TIMEOUT`     | `100`       | The Agent connection timeout (in milliseconds)                              |
+| `DD_TRACE_AGENT_MAX_CONSECUTIVE_FAILURES` | `3`    | IPC-based configurable circuit breaker max consecutive failures             |
 | `DD_TRACE_AGENT_PORT`                | `8126`      | The Agent port number                                                       |
-| `DD_TRACE_AGENT_TIMEOUT`             | `500`       | Maximum time the Agent is allowed to take (in milliseconds)                 |
+| `DD_TRACE_AGENT_TIMEOUT`             | `500`       | The Agent request transfer timeout (in milliseconds)                        |
 | `DD_TRACE_AGENT_CONNECT_TIMEOUT`     | `100`       | Maximum time the allowed for Agent connection setup (in milliseconds)       |
 | `DD_TRACE_ANALYTICS_ENABLED`         | `false`     | Flag to enable trace analytics for relevant spans in web integrations       |
 | `DD_TRACE_DEBUG`                     | `false`     | Enable [debug mode][16] for the tracer                                      |
 | `DD_TRACE_ENABLED`                   | `true`      | Enable the tracer globally                                                  |
 | `DD_TRACE_GLOBAL_TAGS`               | ``          | Tags to be set on all spans: e.g.: `key1:value1,key2:value2`                |
+| `DD_TRACE_REPORT_HOSTNAME`           | `false`     | Enable hostname reporting on the root span                                  |
+| `DD_TRACE_RESOURCE_URI_MAPPING`      | `null`      | CSV of URL-to-resource-name mapping rules; e.g., `/foo/*,/bar/$*/baz`; [see "Custom URL-To-Resource Mapping"][16] |
+| `DD_TRACE_URL_AS_RESOURCE_NAMES_ENABLED` | `false` | Enable URL's as resource names; [see "Map Resource Names To Normalized URI"][17] |
 | `DD_<INTEGRATION>_ANALYTICS_ENABLED` | `false`     | Flag to enable trace analytics for relevant spans in a specific integration |
 
+#### Map Resource Names To Normalized URI
+
+<div class="alert alert-warning">
+This functionality is in public beta. Request early access by contacting <a href="/help">Datadog Support</a>. 
```

Comment:
There is no "early access" required that they need to request. Can we only say "This functionality is in public beta. Contact support if more details are required."